Windows support, or the lack thereof

I was playing around with this repo and a found that the tests are failing on Windows:

<details>
<summary>verbose test report</summary>

```
λ yarn test
yarn run v1.3.2
$ mocha lib/**/*.spec.js && npm run test-typings

  Terminus
    1) runs onSigterm when getting the SIGTERM signal
    2) runs onShutdown after onSigterm
    3) runs onSigint when getting SIGINT signal
    4) runs onShutdown after onSigint
    supports onHealthcheck for the healthcheck route
      √ but keeps all the other endpoints
      √ returns 200 on resolve
      √ returns 503 on reject

  3 passing (1s)
  4 failing

  1) Terminus
       runs onSigterm when getting the SIGTERM signal:

      AssertionError: expected '' to deeply equal 'on-sigterm-runs'
      + expected - actual

      +on-sigterm-runs

      at Context.ex (lib\terminus.spec.js:89:46)

  2) Terminus
       runs onShutdown after onSigterm:

      AssertionError: expected '' to deeply equal 'on-sigterm-runs\non-shutdown-runs'
      + expected - actual

      +on-sigterm-runs
      +on-shutdown-runs

      at Context.ex (lib\terminus.spec.js:100:46)

  3) Terminus
       runs onSigint when getting SIGINT signal:

      AssertionError: expected '' to deeply equal 'on-sigint-runs'
      + expected - actual

      +on-sigint-runs

      at Context.ex (lib\terminus.spec.js:111:46)

  4) Terminus
       runs onShutdown after onSigint:

      AssertionError: expected '' to deeply equal 'on-sigint-runs\non-shutdown-runs'
      + expected - actual

      +on-sigint-runs
      +on-shutdown-runs

      at Context.ex (lib\terminus.spec.js:122:46)

error Command failed with exit code 4.
```

</details>
<br>

I tried to dig into this a bit and found that the whole concept of `SIGTERM` is not supported on Windows.

- https://github.com/nodejs/node-v0.x-archive/issues/5054#issuecomment-15150485

> Short story: you can expect `SIGINT` to work now, as well as `SIGBREAK` and to some extent `SIGHUP`. However `SIGTERM` will never work because killing a process in the task manager is unconditional, e.g. there's no way for an application to detect or prevent it.

Also see https://github.com/nodejs/node/issues/12378.

So can I make a PR which updates the documentation for lack of support on Windows due to inherent OS limitations?

--- a/README.md
+++ b/README.md
@@ -38,7 +38,7 @@ terminus(server, {
   onShutdown,                      // [optional] called right before exiting
 
   // both
-  logger                           // [optional] logger function to be called with errors    
+  logger                           // [optional] logger function to be called with errors
 });
 
 server.listen(PORT);
@@ -78,3 +78,11 @@ terminus(server, {
 
 server.listen(3000);
 ```
+
+## Windows support
+
+Due to inherent platform limitations, `terminus` has limited support for Windows.
+You can expect `SIGINT` to work, as well as `SIGBREAK` and to some extent `SIGHUP`.
+However `SIGTERM` will never work on Windows because killing a process in the task manager is unconditional, i.e., there's no way for an application to detect or prevent it.
+Here's some relevant documentation from [`libuv`](https://github.com/libuv/libuv) to learn more about what `SIGINT`, `SIGBREAK` etc. signify and what's supported on Windows - http://docs.libuv.org/en/v1.x/signal.html.
+Also see https://nodejs.org/api/process.html#process_signal_events.

--- a/README.md
+++ b/README.md
@@ -79,7 +79,7 @@ terminus(server, {
 server.listen(3000);
 ```
 
-## Windows support
+## Limited Windows support
 
 Due to inherent platform limitations, `terminus` has limited support for Windows.
 You can expect `SIGINT` to work, as well as `SIGBREAK` and to some extent `SIGHUP`.

